uint16 servo
uint16 leftMotor
uint16 rightMotor